aten/src/ATen/native/Loss.cpp

```diff
@@ -41,20 +41,18 @@ Tensor cosine_embedding_loss(const Tensor& input1, const Tensor& input2, const T
   auto denom = (mag_square1 * mag_square2).sqrt_();
   auto cos = prod_sum / denom;
 
-  auto zeros = at::zeros_like(cos, LEGACY_CONTIGUOUS_MEMORY_FORMAT);
   auto pos = 1 - cos;
   auto neg = (cos - margin).clamp_min_(0);
-  auto output_pos = at::where(target == 1, pos, zeros);
-  auto output_neg = at::where(target == -1, neg, zeros);
+  auto output_pos = at::where(target == 1, pos, 0);
```

Comment:
I wouldn't bother changing this file in this PR. It could be done separately in a follow-up.

Reply:
Done.